_squirtle/mod.ts

```diff
@@ -70,29 +67,82 @@ const runLogin = async () => {
   await runUser(null, user, env)
 }
 
+const browse = (location: string): Promise<Deno.ProcessStatus> => {
+  return new Promise<Deno.ProcessStatus>((resolve, reject) => {
+    switch (process.platform) {
+      case 'aix':
+      case 'freebsd':
+      case 'linux':
+      case 'openbsd':
+        return Deno.run({
+          cmd: ['gp', 'preview', '--external', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .catch(() =>
+            Deno.run({
+              cmd: ['xdg-open', location],
+              stderr: 'null',
+              stdout: 'null',
+            })
+              .status()
+          )
+          .then(resolve)
+          .catch(reject)
+      case 'darwin':
+        return Deno.run({
+          cmd: ['open', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .then(resolve)
+          .catch(reject)
+      case 'win32':
+        return Deno.run({
+          cmd: ['cmd', '/c', 'start', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .then(resolve)
+          .catch(reject)
+    }
+  })
+    .then((status: Deno.ProcessStatus) => {
+      if (status.success)
+        console.log("opened %s", location)
+      return status
+    })
+}
+
 const runLogout = async (
-  _: any, 
+  _: any,
   internal = false
 ) => {
-  const run1 = Deno.run({
-    cmd: ['gp', 'env', '-u', 'AUTH_TOKEN'],
+  const { AUTH_TOKEN, ENV } = await getEnv()
+  if (!AUTH_TOKEN)
+    return console.log(`Please run the <login> command first`);
+  const run1 = gp({
+    cmd: ['env', '-u', 'AUTH_TOKEN'],
   })
-  const run2 = Deno.run({
-    cmd: ['gp', 'env', '-u', 'ACCESS_TOKEN'],
+  const run2 = gp({
+    cmd: ['env', '-u', 'ACCESS_TOKEN'],
   })
-  const run3 = Deno.run({
-    cmd: ['gp', 'env', '-u', 'CLAIM_TOKEN'],
+  const run3 = gp({
+    cmd: ['env', '-u', 'CLAIM_TOKEN'],
   })
-  const run4 = Deno.run({
-    cmd: ['gp', 'env', '-u', 'REFRESH_TOKEN'],
+  const run4 = gp({
+    cmd: ['env', '-u', 'REFRESH_TOKEN'],
   })
 
   await Promise.all([
-    run1.status(),
-    run2.status(),
-    run3.status(),
-    run4.status(),
-  ])
+    run1,
+    run2,
+    run3,
+    run4,
+  ]).catch()
```

Comment:
Does this work? Can you call catch without a function arg? Might be better to just let the Deno execution catch or else if you really want to ignore maybe `.catch(() => {})` but if this has been tested and works 👍